### PR TITLE
Run Coherence with dotnet.

### PR DESCRIPTION
**Pull request by cesarbs**
cc @pranavkm @victorhurdugaci

### CODE REVIEW

File: src/CoherenceBuild/project.json [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
     "NuGet.Core": "2.10.1",
     "Microsoft.Extensions.CommandLineUtils": "1.0.0-*"
   },
-
-  "commands": {
-    "CoherenceBuild": "CoherenceBuild"
-  },
-
   "frameworks": {
     "dnx451": { }
```
Comment (pranavkm) [inline src/CoherenceBuild/project.json]: `net451`. Doesn't need to be dnx anymore.

File: makefile.shade [hunk 2]
```diff
@@ -40,4 +40,4 @@ use-standard-lifecycle
    }
 
   var args = '--drop-folder ${BUILD_SHARE} --build-branch ${BUILD_BRANCH} --output-path ${BUILD_DIR} --symbols-source-path ${NUGET_SYMBOLS_SHARE} --symbols-output-path ${SYMBOLS_DIR} --symbols-nuget-exe ${SYMBOLS_NUGET_EXE} --nuget-publish-feed ${NUGET_PUBLISH_FEED} --apikey ${APIKEY} --ci-volatile-share ${PACKAGE_DROP_SHARE}'
-  exec program='cmd' commandline='/C dnx CoherenceBuild ${args}' workingdir='${COHERENCE_BUILD_PROJ}'
+  exec program='dotnet' commandline='run -f dnx451 -- ${args}' workingdir='${COHERENCE_BUILD_PROJ}'
```
Comment (pranavkm) [inline makefile.shade]: Do you have to specify `-f dnx451`? Doesn't it infer it?

Comment (cesarbs) [inline makefile.shade]: I thought it would, but no - it complains about not finding a DNX Core target to run.

Comment (pranavkm) [inline makefile.shade]: That's unfortunate. Oh well

Comment (victorhurdugaci) [inline makefile.shade]: What?? That should be a bug

Comment (victorhurdugaci) [inline makefile.shade]: Btw, you're using the standard goals so, at this point, the binaries should be available on disk. Why not just run the exe?

**Comment by muratg** [comment]
@cesarbs Could you try out net451 again?  Or is this done on a separate PR already (in which case, please close this one)

**Comment by victorhurdugaci** [comment]
What's happening with this PR?

**Comment by pranavkm** [comment]
Ooh, I thought the changes from this PR looked familiar. I made this change in another commit - https://github.com/aspnet/Coherence/commit/ae62821eae1a6faf390419e32612e1040f7b2ce8

**Comment by muratg** [comment]
@pranavkm So this one good to close?

**Comment by pranavkm** [comment]
Yup.

**Pull request closed without merging**